### PR TITLE
added the Happy Recovery Story page

### PR DESCRIPTION
**Pull request by anagha-chaudhari**
## Description
As stated in the issue #495 I have added a page that showcases the rescuers and the recovery stories of the rescued animals

## Related Issue
[Cite any related issue(s) this pull request addresses. If none, simply state "None"]

## Screenshots / GIFs (if applicable)
paws1

https://github.com/Innovateninjas/Paws-frontend/assets/143149376/e8572fa5-8df8-48a0-972d-34672f301309

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
I have made the folder under src / pages / user / RecoveredsPage
The required images are added in the images folder of the RecoveredsPage folder.
Please review the PR, and if there are any changes required, let me know

### CODE REVIEW

**Comment by codewarnab** [comment]
good work but in the issue comments i mentioned that it will be added in landing page which is not ready , so do one thing just create a separate route for this no need to add any tab in the nav bar in the exisiting ui

**Comment by anagha-chaudhari** [comment]
Hello @codewarnab, I'm sorry for my mistake. I have removed the tab from the navbar, and the path to the page is **/recovereds-page** Please review once again

**Comment by anagha-chaudhari** [comment]
@codewarnab please review

File: src/Components/shared/Navbar.jsx [hunk 1]
```diff
@@ -25,57 +23,57 @@ const Navbar = () => {
...
-              showLabel
-              label = "Home"  
+                key="home" value="/" component={Link} 
+                to="/" icon={<MdHome size={30}/>} 
+                showLabel
+                label="Home"  
               />,
               <BottomNavigationAction key="register" value="/register" showLabel label="Register" component={Link} to="/register" icon={<FaUserPlus size={28} />} />,
               <BottomNavigationAction key="login" value="/login" showLabel label="Login" component={Link} to="/login" icon={<FaUser size={24} />} />,
+              /*<BottomNavigationAction key="RecoveredsPage" value="/recovereds-page" showLabel label="Stories" component={Link} to="/recovereds-page" icon={<FaPaw size={28} />} />, // for recovered stories section*/
```
Comment (codewarnab) [inline src/Components/shared/Navbar.jsx:70]: remove this part

File: src/Components/shared/Navbar.jsx [hunk 2]
```diff
@@ -2,14 +2,12 @@ import React, { useState, useEffect } from 'react';
 import { Link, useLocation } from 'react-router-dom';
 import BottomNavigation from '@mui/material/BottomNavigation';
 import BottomNavigationAction from '@mui/material/BottomNavigationAction';
-import { FaUser, FaUserPlus} from 'react-icons/fa';
+import { FaUser, FaUserPlus, FaPaw } from 'react-icons/fa';
```
Comment (codewarnab) [inline src/Components/shared/Navbar.jsx:5]: remove this icon

File: src/Components/shared/Navbar.jsx
Comment (codewarnab) [inline src/Components/shared/Navbar.jsx:1]: if possible revert this file as before

**Review by codewarnab — CHANGES_REQUESTED** [review]

**Comment by anagha-chaudhari** [comment]
I have made the changes and added the original initial code to Navbar.jsx, please check @codewarnab

**Comment by anagha-chaudhari** [comment]
I have done changes and added the current original code of navbar to Navbar.jsx, please check @codewarnab

**Review by codewarnab — APPROVED** [review]

**Pull request merged**